website/content/docs/agent-and-proxy/proxy/apiproxy.mdx

```diff
@@ -6,7 +6,7 @@ description: >-
   for Vault's API.
 ---
 
-# Vault proxy API proxy
+# Vault Proxy API proxy
```

Comment:
This is a little redundant. Simplify to "Vault Proxy" and the following text could also be simplified to "Vault Proxy provides the ability to make API calls to the Vault Server"

Reply:
This is describing the API Proxy subsystem of Proxy. While it's the 'main' subsystem, Proxy has a lot of functionality and I think it makes sense to have a page to focus on how Proxy's API proxy works

Reply:
ah ok, thanks!